frontend/src/resources/cluster-curator.ts

```diff
@@ -56,6 +56,7 @@ export interface ClusterCurator {
 export interface ClusterCuratorAnsibleJob {
     name: string
     extra_vars?: Record<string, string>
+    type?: string
```

Comment:
Let's be specific on the type here. In my PR for the operator detection, this is how I set up the types: https://github.com/stolostron/console/pull/2313/files#diff-628bf0cfb854ec47885ef6e2f08a9a20d41dab04cc0c9a37ac1333aafdf93be4